<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TerParaSetting</class>
 <widget class="QWidget" name="TerParaSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>100000</width>
    <height>100000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>终端参数设置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="TRES.qrc">
    <normaloff>:/new/prefix1/Tres/Gear.ico</normaloff>:/new/prefix1/Tres/Gear.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>460</y>
     <width>381</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="sureBttn">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="cancelBttn">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="uGrpBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>621</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>11</y>
      <width>356</width>
      <height>89</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>主IP：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="mIpLed">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>25</height>
        </size>
       </property>
       <property name="inputMask">
        <string>000.000.000.000</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>主端口号：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="mPortLed">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>25</height>
        </size>
       </property>
       <property name="inputMask">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>备IP：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="bakIPLed">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>25</height>
        </size>
       </property>
       <property name="inputMask">
        <string>000.000.000.000</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>备端口号：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="bakPortLed">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>25</height>
        </size>
       </property>
       <property name="inputMask">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>APN：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="apnLed">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>25</height>
        </size>
       </property>
       <property name="inputMask">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="uFileLed">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>500</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="asUpFileBttn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>指定升级文件</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="swReleaseTLed">
    <property name="geometry">
     <rect>
      <x>471</x>
      <y>74</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>25</height>
     </size>
    </property>
    <property name="inputMask">
     <string>00:00:00</string>
    </property>
    <property name="text">
     <string>::</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>381</x>
      <y>74</y>
      <width>84</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>软件发布时间：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="swVersionLed">
    <property name="geometry">
     <rect>
      <x>471</x>
      <y>12</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>25</height>
     </size>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>381</x>
      <y>43</y>
      <width>84</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>软件发布日期：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>381</x>
      <y>12</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>软件版本：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="swReleaseDLed">
    <property name="geometry">
     <rect>
      <x>471</x>
      <y>43</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>25</height>
     </size>
    </property>
    <property name="inputMask">
     <string>0000-00-00</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="uParaChbx">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>升级参数</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="spParaChbx">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>规约类参数</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="pGrpBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>621</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>179</width>
      <height>74</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>终端上行规约：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="TerUpPcbx">
       <item>
        <property name="text">
         <string>GB09</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GB13</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>维护</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CSG2015</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>主站规约：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="masterPCbx">
       <item>
        <property name="text">
         <string>新联</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>无</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>电表上行规约：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="meterUPCbx">
       <item>
        <property name="text">
         <string>DTL1997</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>190</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>升级要求：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="terAddrAssCbx">
       <item>
        <property name="text">
         <string>无地址指定</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>地址指定</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>终端指定模式：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="upFiledemandLed">
       <item>
        <property name="text">
         <string>湖北</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>山东</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>南网</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新联</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>普通</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="terAddrCCbx">
       <item>
        <property name="text">
         <string>2字节</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4字节</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>南网3字节</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>终端地址字节数：</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>214</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>升级文件段大小：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="FileSSLed">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="inputMask">
        <string>0000000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>字节</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QCheckBox" name="miscParaChbx">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>杂项参数</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="mGrpBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>621</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>专变类型：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="speTerTypeCbx">
       <item>
        <property name="text">
         <string>三相四线220V</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>三相三线100V</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>三相四线57V</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>154</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>终端类型：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="terTypeCbx">
       <item>
        <property name="text">
         <string>专变终端</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>I型集中器</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>II型集中器</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>331</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QCheckBox" name="overTimeChBx">
       <property name="text">
        <string>开启单帧超时重传 重传间隔：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="overTimerLed">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>25</height>
        </size>
       </property>
       <property name="inputMask">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>ms</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>212</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string> 收发延时：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="SRDelayLed">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>25</height>
        </size>
       </property>
       <property name="inputMask">
        <string>0000000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>ms</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="TRES.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>spParaChbx</sender>
   <signal>clicked(bool)</signal>
   <receiver>pGrpBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uParaChbx</sender>
   <signal>clicked(bool)</signal>
   <receiver>uGrpBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>miscParaChbx</sender>
   <signal>clicked(bool)</signal>
   <receiver>mGrpBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
